doc/api/repl.md

```diff
@@ -424,7 +483,20 @@ reference to the `context` object as the only argument.
 This can be used primarily to re-initialize REPL context to some pre-defined
 state:
 
-```js
+```mjs
+import repl from 'node:repl';
+
+function initializeContext(context) {
+  context.m = 'test';
+}
+
+const r = repl.start({ prompt: '> ' });
+initializeContext(r.context);
+
+r.on('reset', initializeContext);
+```
```

Comment:
I don't think this one is working as it's described? Maybe I'm misunderstanding what's supposed to happen here? 
<img width="1102" alt="image" src="https://github.com/user-attachments/assets/4dba5c00-f771-4895-aa53-2a2df864e10f">

Reply:
It resets the context, `m` equals to `'test'`, but if you were to do something like `m = 'other thing'` and then you call `.clear` it will reset the context, so when you print the value of `m` it would be reset back to `'test'` :blush: